### ISSUE TITLE
How do I correct line recognition?

### ISSUE THREAD

**Issue by zabak** [DCGM/pero_ocr_web#33]
When correcting OCR for National Museum, the regions were mostly recognized without problems. When correcting OCR, however, I found cases where part of word was not detected as part of line. Sometimes, it has not been detected at all:
![image](https://user-images.githubusercontent.com/811537/81814363-f9c6b080-9528-11ea-96fe-491191e626bc.png)
Other times: there were overlapping or duplicate detections:
![image](https://user-images.githubusercontent.com/811537/81814830-81142400-9529-11ea-9e36-d6ebd872f401.png)
How should I treat it when correcting?

**Comment by zabak** [comment]
This is really a problem - please implement support for fixing the text line boundaries and use them to make a better model.

**Comment by zabak** [comment]
see also https://pero-ocr.fit.vutbr.cz/document/collaborators/13e60e04-734b-464a-b63d-eceb8d0a2563

**Comment by michal-hradis** [comment]
We are dubugging layout editor. Normally, you edit text in /ocr/show_results/... The new interface is running at  /ocr/show_results_new/... Just manually change the URL. I'll probably have to explain how the interface works.

**Comment by zabak** [comment]
Yes please, I will need some explanation, the interface is not intuitive. Default zoom should zoom to fill the window width or height, not zoom out so much as it does now. There is no way to delete a row. I have no idea how to select two rows to join them. How to resize a row, how edit a shape of a region.

**Comment by zabak** [comment]
@michal-hradis please add the explanation here. Also, how to run OCR again without losing the baseline information.

**Comment by michal-hradis** [comment]
Text transcriptions can be generated again and again without any loss of manual text corrections. Text line detection can not be repeated without loosing manual corrections.

How to edit text lines:
1. Select by left click.
2. Press CTRL to show controll points. 
3. Drag controll points with mouse left button. You can delete controll points by moving the line end over them. You can't add controll points.
4. Change line height by dragging "top" and "bottom" controll points et the end and beginnig of a line.

Delete line: 
1. Right click on a line.
2. Slect delete in the context menu, but check carefully that it shows:  "Delete row" and not "Delete region" - the second one shows by mistake if you move the mouse when clicking.

Add lines:
1. Select a region.
2. Slect tool: "Create new row (baseline). 
3. Create a baseline by left clicking. Finish baseline by pressing enter or right click.
4. Select line hight - top part and bottom part separately. Use left click to set the heights.

Edit regions:
1. Select region.
2. Press alt to show region controll points.
3. Drag the controll points around.

You can further:
- Add regions 
- Delete regions (deletes all associated text lines)
- Merge text lines (does not work properly at the moment)
